dev/calc_allale_depths_cram.py

```diff
@@ -0,0 +1,88 @@
+import pysam
```

Comment:
Can you correct the spelling mistake in allele in the script name?

Reply:
Ah, good catch! However, forgot to mention that this script is nowhere near ready. It was just the strelka2 script that I wanted eyes on, but not necessarily to put into prod. Just to validate my thoughts. However, while working on this script, I may want to change some logic in that one too...